tests/tools/oesign-engine/test-enclave/enclave/test-enclave-kss.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Enclave settings:
+Debug=1
+Kss=1
+NumHeapPages=1024
+NumStackPages=1024
+NumTCS=1
+ProductID=1
+SecurityVersion=1
+IsvFamilyID=7cc330ff-c8cf-49d4-92ef-e1674794f820
+IsvExtProductID=8cc330ff-c8cf-49d4-92ef-e1674794f820
```

Comment:
Linux apps generally expect a newline character at the end of each text file for correct parsing, so GitHub flags the missing newline with the symbol you see here. Please try to add the required newlines back in when you see it in your PRs.

Reply:
Removed per refactoring.